pypeit/scripts/coadd_2dspec.py

```diff
@@ -140,6 +140,9 @@ def main(args):
         else:
             basename = args.basename
 
+
+        # TODO Heliocentric for coadd2d needs to be thought through. Currently turning it off.
```

Comment:
Should we add a comment in `coadd2d.rst` ?
  seems like it

Reply:
Well the coadd uses the individual waveimgs, so as long as heliocentric corrections are being done on those, I think this should work. However, I admittedly have not thought this through carefully which was the reason for the comment. Maybe we should make this an issue on GitHub.